Dockerfile

```diff
@@ -39,7 +39,10 @@ RUN set -eux ; \
         make \
         pkgconf \
         file \
-        curl-dev; \
+        curl-dev \
+        libpng \
```

Comment:
This should be where is not used - -virtual .fetch-deeps.    Beacouse libpng will be removed where is clean up comment. Line 132

Reply:
Fixed in  next PR. Libraries have been moved to first apk add to avoid being deleted

Reply:
Dev package can be where is - -virtual .fetch-deeps.   Because this can be safely removed after building php.